README.md

```diff
@@ -60,60 +60,107 @@ Want to override the default theme of Chroma? No problem!
 
 1. Create your component-level overrides and palette overrides. Chroma leverages Material-UI's global theme variation [to override specific component styles](https://material-ui.com/customization/components/#5-global-theme-variation).
 
-    ```js
-    // theme.ts
-    import {
-      createPalette,
-      createTheme,
-      Theme,
-    } from '@lifeomic/chroma-react/styles';
-    import { Overrides } from '@lifeomic/chroma-react/styles/overrides';
-
-    // The overrides specified here will be *global* component overrides!
-    export const overrides = (theme: Theme): Overrides => ({
-      ChromButton: {
-        root: {
-          background: 'red',
-        },
-        outlined: {
-          border: '1px solid red',
-        },
-      },
-    });
-
-    export const palette = createPalette({
-      primary: {
-        main: '#02bff1',
-      },
-    });
-
-    export const theme = createTheme({
-      overrides,
-      palette,
-    });
-    ```
+   ```js
+   // theme.ts
+   import {
+     createPalette,
+     createTheme,
+     Theme,
+   } from '@lifeomic/chroma-react/styles';
+   import { Overrides } from '@lifeomic/chroma-react/styles/overrides';
+
+   // The overrides specified here will be *global* component overrides!
+   export const overrides = (theme: Theme): Overrides => ({
+     ChromButton: {
+       root: {
+         background: 'red',
+       },
+       outlined: {
+         border: '1px solid red',
+       },
+     },
+   });
+
+   export const palette = createPalette({
+     primary: {
+       main: '#02bff1',
+     },
+   });
+
+   export const theme = createTheme({
+     overrides,
+     palette,
+   });
+   ```
 
 2. Update your theme provider.
 
-    ```jsx
-    import { ThemeProvider } from '@lifeomic/chroma-react/styles';
-    import { theme } from './theme';
+   ```jsx
+   import { ThemeProvider } from '@lifeomic/chroma-react/styles';
+   import { theme } from './theme';
 
-    function App({ children }) {
-      return <ThemeProvider theme={theme}>{children}</ThemeProvider>;
-    }
-    ```
+   function App({ children }) {
+     return <ThemeProvider theme={theme}>{children}</ThemeProvider>;
+   }
+   ```
 
 ## Importing Component Styles Only?
 
-    Need to build a custom component, but want to use the styles hook of an existing Chroma component?
+Need to build a custom component, but want to use the styles hook of an existing Chroma component?
+
+```jsx
+import { useStyles } from '@lifeomic/chroma-react/components/Button/Button';
+
+const CustomButton = ({}) => {
+  const classes = useStyles({});
+  return <button className={classes.root}>Custom Button</button>;
+};
```

Comment:
This section in particular had a bad backtick block due to the spacing, so it's fixed with this.